code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm

```diff
@@ -284,74 +284,3 @@
 	if(!can_use_ability(target, override_flags = XACT_IGNORE_SELECTED_ABILITY))
 		return ..()
 	return TRUE
-
-// ***************************************
-// *********** Rip limb
-// ***************************************
-
-// Called when pulling something and attacking yourself with the pull
-/mob/living/carbon/xenomorph/proc/pull_power(mob/M)
-	if (isxenowarrior(src) && !ripping_limb && M.stat != DEAD)
-		ripping_limb = TRUE
-		if(rip_limb(M))
-			stop_pulling()
-		ripping_limb = FALSE
```

Comment:
do you not need to remove the `ripping_limb ` define too, or is it used for something else?